src/main/java/com/sparrowwallet/sparrow/AppController.java

```diff
@@ -149,6 +149,12 @@ public class AppController implements Initializable {
     private CheckMenuItem hideEmptyUsedAddresses;
     private static final BooleanProperty hideEmptyUsedAddressesProperty = new SimpleBooleanProperty();
 
+    @FXML
+    private CheckMenuItem hideAmounts;
+
+    @FXML
+    private ToggleButton hideAmountsToggle;
```

Comment:
I'm not convinced this feature warrants the visual complexity of adding a status bar icon.